index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="stylesheet" href="styles/navbar.css">
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="styles/footer.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <!-- nav area  -->
+   <header class="header">
+    <nav class="nav-area">
+        <ul>
+            <a href="#"><li>Works</li></a>
+            <a href="#"><li>Blog</li></a>
+            <a href="#"><li>Contact</li></a>
+        </ul>
+    </nav>
+   </header>
+   <!-- first section  -->
+   <div class="first-frame">
```

Comment:
Use appropriate names for classes.

```suggestion
   <div class="hero-container">
```